docs/caching2.md

```diff
@@ -0,0 +1,112 @@
+## Caching
+
+There are two different layers of caching on Artsy.net, namely:
+
+- Caching at the Relay query level, which happens as users navigate from page to page on the client. This is an entirely client-side cache, and typically configured via the route. We will refer to this going forward as 'Relay caching'.
+- Caching at the [Metaphysics CDN](https://github.com/artsy/metaphysics/blob/1b3940a2387ab0bf308c2ca7b597e40fad015b4d/docs/cdn.md) level - which has the effect of caching the entire response for a given query. We will refer to this going forward as 'CDN caching'.
+
+For both of these caching layers, there are several different configurations that can be used/combined to achieve desired behavior.
+
+### General Cache Behavior
+
+#### Logged out:
+
+By default, we typically cache (at both layers) all queries for logged-out visitors, unless otherwise opted-out with `cacheConfig: { force: true }`. There is also a `serverCacheTTL: 0` - which has the effect of skipping the CDN cache, (so the initial request will always hit Metaphysics) - but subsequently Relay caching will be enabled. There are also certain client-side queries made using Relay's `fetchQuery` - which implicitly _adds_ a `force: true` configuration to the query, thus skipping the cache as well. For queries made using `fetchQuery` - you need to add the `force: false` directive explicitly in order to cache. The important thing to remember is: **everything is cached by default unless opted-out (and there are two ways to opt-out, both layers via `force: true`, or just the CDN layer via `serverCacheTTL: 0`)**.
```

Comment:
```suggestion
By default, we typically cache (at both layers) all queries for logged-out visitors, unless otherwise opted-out with `cacheConfig: { force: true }`. There is also a `serverCacheTTL: 0` (set on the route config) which has the effect of skipping the CDN cache so the initial request will always hit Metaphysics, while leaving client-side caching enabled. There are also certain client-side queries made using Relay's `fetchQuery` - which implicitly _adds_ a `force: true` configuration to the query, thus skipping the cache as well. For queries made using `fetchQuery` - you need to add the `force: false` directive explicitly in order to cache. The important thing to remember is: **everything is cached by default unless opted-out (and there are two ways to opt-out, both layers via `force: true`, or just the CDN layer via `serverCacheTTL: 0`)**.
```